Dockerfile

```diff
@@ -33,7 +33,7 @@ LABEL maintainer="Microsoft" \
 # jq - we include jq as a useful tool
 # pip wheel - required for CLI packaging
 # jmespath-terminal - we include jpterm as a useful tool
-RUN apk add --no-cache bash openssh ca-certificates jq curl openssl git zip \
+RUN apk update && apk add --no-cache bash openssh ca-certificates jq curl openssl git zip \
```

Comment:
The apk index on the base image doesn't have the deps I added, so need to update it.

Reply:
Hmm, I just realized what the --no-cache does. Would you prefer I don't apk update here, and instead add --no-cache to the .netcoredeps line? Or alternatively just install those deps in this command?